Bio/AlignIO/NexusIO.py

```diff
@@ -114,7 +114,10 @@ def write_alignment(self, alignment):
         #For larger alginments, interleave to avoid very long lines
         #in the output - something MrBayes can't handle.
         #TODO - Default to always interleaving?
-        n.write_nexus_data(self.handle, interleave=(columns > 1000))
+        if(interleave):
```

Comment:
@PamelaM's suggestion made sense to me too. It also solve the style issue with C like ``if(interleave):`` which should have been just ``if interleave:``